lib/v2/components/profile_avatar.dart

```diff
@@ -47,7 +49,11 @@ class ProfileAvatar extends StatelessWidget {
           : account.substring(0, 2).toUpperCase();
 
       return Container(
-        color: AppColors.getColorByString(shortName),
```

Comment:
why this needed? I thought this widget was completed and working well?

Reply:
It was but not when the size is less than 40.